pkg/pillar/types/zedroutertypes.go

```diff
@@ -1065,3 +1068,62 @@ type AppBlobsAvailable struct {
 type AppInfo struct {
 	AppBlobs []AppBlobsAvailable
 }
+
+// AppMACGenerator persistently stores ID of the MAC generator that was used to generate
+// MAC addresses for interfaces of a given app.
+type AppMACGenerator struct {
+	*UuidToNum
+}
+
+// New is used by objtonum.ObjNumPublisher.
+func (g *AppMACGenerator) New(objKey objtonum.ObjKey) objtonum.ObjNumContainer {
+	return &AppMACGenerator{
+		UuidToNum: g.UuidToNum.New(objKey).(*UuidToNum),
```

Comment:
Looks like a yetus thing to fix.
I think elsewhere we have explicit log.Fatal for conditions like this.

Reply:
Fixed